packages/react-core/src/components/Button/examples/ButtonCount.tsx

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import { Button } from '@patternfly/react-core';
+
+export const ButtonCount: React.FunctionComponent = () => {
```

Comment:
Nit: the exported name could match the example title a bit more, e.g. `export const ButtonWithCount`.